From: Max Filippov <jcmvbkbc@gmail.com>
Date: Wed, 26 Feb 2020 12:43:52 -0800
Subject: target/xtensa: fix pasto in pfwait.r opcode name

Git-commit: 1a03362b14affa4d8ddede55df6e21d7a07b87c2

Core xtensa opcode table has pfwait.o instead of pfwait.r. Fix that.

Fixes: c884400f2988 ("target/xtensa: implement block prefetch option opcodes")
Signed-off-by: Max Filippov <jcmvbkbc@gmail.com>
Signed-off-by: Bruce Rogers <brogers@suse.com>
---
 target/xtensa/translate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/xtensa/translate.c b/target/xtensa/translate.c
index a99f5296e2f4f1d01fd21cfb3fd0..2ec0e5a047ed0e9b9c0926518649 100644
--- a/target/xtensa/translate.c
+++ b/target/xtensa/translate.c
@@ -3740,7 +3740,7 @@ static const XtensaOpcodeOps core_ops[] = {
         .name = "pfwait.a",
         .translate = translate_nop,
     }, {
-        .name = "pfwait.o",
+        .name = "pfwait.r",
         .translate = translate_nop,
     }, {
         .name = "pitlb",
